cmds/proxy_server/main.go

```diff
@@ -0,0 +1,130 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/gorilla/mux"
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+	"github.com/threefoldtech/grid_proxy_server/explorer"
+)
+
```

Comment:
Delcare a global variable here e.g GitCommit and link it to version flag and add to your build pipeline like that 

```
export GIT_COMMIT=$(git rev-list -1 HEAD) && \
  go build -ldflags "-X main.GitCommit=$GIT_COMMIT"
```

and for docker file update to

```
RUN GIT_COMMIT=$(git rev-list -1 HEAD) && \
  go build -ldflags "-X main.GitCommit=$GIT_COMMIT"
```